### ISSUE TITLE
Support Chinese attack？

### ISSUE THREAD

**Issue by SueeH** [QData/TextAttack#799]
#719 shows QData support chinese attack, the code merged to master branch？
if so， how to use chinese attack or augmentation？

**Comment by qiyanjun** [comment]
Yes — Chinese support is merged and on \`master\` (\`textattack.attack_recipes.ChineseRecipe\`, using \`jieba\` for segmentation and Chinese-specific transformations like homophone/morphonym character swaps). To use it:

\`\`\`python
from textattack.attack_recipes import ChineseRecipe
attack = ChineseRecipe.build(model_wrapper)
\`\`\`

There's also a Chinese example notebook at \`docs/2notebook/Example_6_Chinese_Attack.ipynb\`. Note this covers *attacks*, not augmentation — there's no dedicated Chinese-language augmentation recipe currently (augmentation is English-focused), just the attack recipe.

**Comment by qiyanjun** [comment]
Answered above — Chinese attack support is merged and usable via `ChineseRecipe`, with a usage example and pointer to the example notebook. Closing as answered.
